src/stanc/stanc.ml

```diff
@@ -381,4 +80,4 @@ let () =
   | Error internal_error ->
       Out_channel.output_string stderr internal_error;
       Out_channel.flush stderr;
-      exit 2
+      exit 125
```

Comment:
These changes to exit codes need a bit more explanation.

Reply:
125 is what Cmdliner raises for unexpected errors: https://erratique.ch/software/cmdliner/doc/Cmdliner/Cmd/Exit/index.html#val-internal_error

because we’re passing catch:false, we’re not obligated to do exactly the same thing, so I’m happy to do whatever. I don’t expect anyone is relying on the exact exit code of the compiler when it ICEs, and I don’t think we had previously documented it anyway